ql/src/java/org/apache/hadoop/hive/ql/ddl/table/misc/truncate/TruncateTableAnalyzer.java

```diff
@@ -116,10 +116,13 @@ private void checkTruncateEligibility(ASTNode ast, ASTNode root, String tableNam
 
   private void addTruncateTableOutputs(ASTNode root, Table table, Map<String, String> partitionSpec)
       throws SemanticException {
-    boolean truncateKeepsDataFiles = AcidUtils.isTransactionalTable(table) &&
-        MetastoreConf.getBoolVar(conf, MetastoreConf.ConfVars.TRUNCATE_ACID_USE_BASE);
+    boolean truncateUseBase = (HiveConf.getBoolVar(conf, HiveConf.ConfVars.HIVE_ACID_TRUNCATE_USE_BASE)
+        || HiveConf.getBoolVar(conf, HiveConf.ConfVars.HIVE_ACID_LOCKLESS_READS_ENABLED))
```

Comment:
What happened with the `TRUNCATE_ACID_USE_BASE` config?
Do we need to deprecate it? Was this used for the same?

Reply:
It didn't work (only in tests). Not sure if we need to deprecate old property.

Reply:
If it was never working, we do not need to deprecate it